examples/Makefile

```diff
@@ -363,10 +410,25 @@ pubsub-nodejs-verify:
 
 pubsub-nodejs-update:
 	kubeless topic create s3-nodejs-2 || true
-	kubeless function update pubsub-nodejs --trigger-topic s3-nodejs-2
+	kubeless trigger kafka update pubsub-nodejs --trigger-topic s3-nodejs-2
```

Comment:
lets split trigger tests into a clear section in which we test everything (with just one runtime language, we don't need to do that for every language). We need tasks for:
 - create (cronjob|http|kafka)trigger
 - verify-create
 - update (cronjob|http|kafka)trigger
 - verify-update
 - delete (cronjob|http|kafka)trigger
 - verify delete

Reply:
thinking better about this, let's do this in a different PR

Reply:
Sure, restructuting and hook it in various CI jobs will take time. Lets make it in seperate PR.